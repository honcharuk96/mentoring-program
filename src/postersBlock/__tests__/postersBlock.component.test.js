import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import PostersArea from '../postersBlock.component';

const shallow = new ShallowRenderer();
describe('<PostersArea />', () => {
  it('should render PostersArea', () => {
    const header = shallow.render(<PostersArea />);
    expect(header).toMatchSnapshot();
  });
});
