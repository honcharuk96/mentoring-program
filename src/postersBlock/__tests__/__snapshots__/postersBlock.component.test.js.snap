// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostersArea /> should render PostersArea 1`] = `
<React.Fragment>
  <globalStyles__Main>
    <Memo(Connect(Component)) />
    <Memo(Connect(Component)) />
    <Memo(Connect(ListPosters)) />
  </globalStyles__Main>
</React.Fragment>
`;
